tests/fixtures/publishers/trigger-parameterized-builds.yml

```diff
@@ -0,0 +1,62 @@
+- job:
+    name: trigger-parameterized-builds
+    project-type: freestyle
+    publishers:
+      - trigger-parameterized-builds:
+        - condition: ALWAYS
+          current-parameters: false
+          node-parameters: false
+          project:
+            - other_job
+            - foo
+            - bar
+          predefined-parameters: |
+            foo=bar
+            bar=foo
+          trigger-from-child-projects: false
+          trigger-with-no-params: false
+        - condition: ALWAYS
```

Comment:
It would be good to leave a comment which would describe the test case purpose (like `git-revision` here) or put each test case in a separate file with a self-descriptive name.